public/video-ui/src/services/UploadsApi.js

```diff
@@ -0,0 +1,41 @@
+import {pandaReqwest} from './pandaReqwest';
+
+// See http://andrewhfarmer.com/aws-sdk-with-webpack/ for why this is strange
+import 'aws-sdk/dist/aws-sdk';
+const AWS = window.AWS;
```

Comment:
💩 💩 💩 💩 💩 💩 

Any suggestions greatly appreciated? My Webpack knowledge is not sufficient